test/Kestrel.Performance/Kestrel.Performance.csproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="..\..\build\common.props" />
 
   <PropertyGroup>
-    <AssemblyName>Microsoft.AspNetCore.Server.Kestrel.Performance</AssemblyName>
```

Comment:
Benchmarks fail to run if the name isn't changed https://github.com/dotnet/BenchmarkDotNet/issues/498